README.md

```diff
@@ -29,26 +29,56 @@ By default, the prefix is `/lambda/mipsSecret`, resulting the following required
 
 ### Template Parameters
 
+The following template parameters are used to configure CloudFormation resources
+
+| Template Parameter | Description |
+| SsmAliasPrefix | Prepend this value to the Alias Name of the KMS key created. |
+| SsmKeyAdminArn | Grant KMS admin access to these ARNs for managing secure secrets. |
+
 The following template parameters are set as environment variables in the lambda environment:
+
 | Template Parameter | Environment Variable | Description |
 | --- | --- | --- |
-| CacheTTL | CacheTTL | Value for `max-age` in the `cache-control` header |
-| SsmParamPrefix | SsmPath | Path prefix for secure parameters |
-| MipsOrganization | MipsOrg | Log in to this organization in the finance system |
+| CacheTTL | CacheTTL | TTL value for CloudFront cache objects. |
+| MipsOrganization | MipsOrg | Log in to this organization in the finance system. |
+| SsmParamPrefix | SsmPath | Path prefix for required secure parameters. |
+| CodesToOmit | CodesToOmit | List of numeric codes to treat as inactive. |
+| CodesToAdd | CodesToAdd | List of "code:name" strings to add to the active list. |
 
 ### Triggering
 
-The CloudFormation template will output the endpoint URL that can be loaded in a browser, e.g.:
-`https://abcxyz.cloudfront.net/all/costcenters.json`
+The CloudFormation template will output all available endpoint URLs for triggering the lambda, e.g.:
+`https://abcxyz.cloudfront.net/accounts`
+`https://abcxyz.cloudfront.net/tags`
```

Comment:
information on where to find these endpoints in AWS would be helpful.

Reply:
i guess you didn't want to do this?

Reply:
sorry, I thought I addressed that with another sentence in this section, but maybe I didn't understand what you meant